src/include/dhandle.h

```diff
@@ -104,7 +104,14 @@ struct __wt_data_handle {
     WT_DSRC_STATS *stats[WT_COUNTER_SLOTS];
     WT_DSRC_STATS *stat_array;
 
-/* Flags values over 0xff are reserved for WT_BTREE_* */
+/*
+ * Flags values over 0xfff are reserved for WT_BTREE_*. This lets us combine the dhandle and btree
+ * flags when we need, for example, to pass both sets in a function call.
+ *
+ * To help avoid accidental overrun of the flag values, we add a special flag value that should
+ * always be the last and highest. At runtime, we use this value to assert that the dhandle flags
+ * haven't run into the space reserved for btree flags.
```

Comment:
If you use the preprocessor, change the comment to not say 'at runtime'.

Reply:
Done